### ISSUE TITLE
Pagination and search examples

### ISSUE THREAD

**Issue by hrvoje-grabusic** [Kooboo/Kooboo#222]
Hi,

I can't find in the documentation or the examples sites how to use search index and create a pager.

Is this possible with k-script ?

Is there a blog site example with search and pagination ?

**Comment by nhaberl** [comment]
yes, paging would be great and also how to use native sql to retrieve data
